Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeVaultRequest.swift

```diff
@@ -0,0 +1,27 @@
+import BraintreePayPal
+
+/**
+ Options for the PayPal Vault flow.
+ */
+@objc public class BTPayPalNativeVaultRequest: BTPayPalVaultRequest, BTPayPalNativeRequest {
```

Comment:
Is this class needed? I thought this flow was only for checkout requests.

Reply:
+1 To this question. I was under the assumption vault & all flows should work with the Native XO, but I know Susan saw some errors when testing Billing Agreement. So, maybe Native XO isn't compatible with vault yet?

Reply:
I think we need to create a demo app so that we can test this. But what we heard from the Native Checkout team is that you can request a billing agreement and it will fall back to a web flow.